app/tests/unit/pages/nanomine/dynamfitresult.spec.js

```diff
@@ -0,0 +1,30 @@
+import createWrapper from '../../../jest/script/wrapper'
+import DynamfitResult from '@/pages/nanomine/tools/dynamfitResult/DynamfitResult.vue'
+
+var wrapper = null
+global.console = {
+  log: jest.fn(), // console.log are ignored in tests
+
+  // Keep native behavior for other methods
+  error: console.error,
+  warn: console.warn,
+  info: console.info,
+  debug: console.debug
+}
+global.fetch = jest.fn()
```

Comment:
Can we make this re-usable if possible

Reply:
Do we have a file for test configuration? It doesn't look like [test-setup.js](https://github.com/Duke-MatSci/materialsmine/blob/2f2edf54aa4b62b8b1203bd62294af9097367ea5/app/tests/jest/script/test-setup.js) is being called/used yet, should I add the global overrides to that and add it as a jest setup file?

Reply:
Totally agree

Reply:
Great. Did that in 41e5d77. All current tests still run, but I'll bring up the change at the standup tomorrow just to let everyone know. Then I'll merge the PR.